src/Altinn.Apps/KubernetesWrapper/deployment/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
Could you write a more describing description? This is not a chart for Kubernetes, the helm chart for the Kubernetes API wrapper.

Reply:
Fixed